### PR TITLE
Create staking_sTLOS.md

### PR DESCRIPTION
**Pull request by HermanusKB007**
On staking sTLOS - this feature has not yet been deployed to the public yet. After it has been deployed, this needs to get updated.

### CODE REVIEW

**Review by HermanusKB007 — COMMENTED** [review]
Updated docs

**Comment by nathanduft44** [comment]
Still need to add a few photos

File: docs/learn/defi_guide/staking_sTLOS.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Staking sTLOS
+
+As a single-asset staking token, sTLOS mimics the REX-style token staking on Telos native, which does not suffer impermanent loss (as seen in dual-asset staking pools) and therefore presents a far lower financial risk for investors.
+
+sTLOS can be staked on Teloscan. Staked sTLOS provides access to a steady income and our DeFi applications.
```
Comment (nathanduft44) [inline docs/learn/defi_guide/staking_sTLOS.md:5]: TLOS can be staked for sTLOS receipt token

**Review by HermanusKB007 — COMMENTED** [review]
Updated text to reflect the difference between TLOS and sTLOS better.

**Comment by HermanusKB007** [comment]
Updated the docs

**Pull request merged**